Sample R notebook.ipynb

```diff
@@ -24,7 +24,7 @@
    "outputs": [],
    "source": [
     "# Defining a\n",
-    "a <- seq(-10 * pi, 10 * pi, 0.1)"
+    "a <- seq(-5 * pi, 5 * pi, 0.05)"
```

Comment:
poti sa faci commenturi si pe asta, desi e mai urat
aia e

Reply:
apar si pe github

Reply:
Eh, aia e, mai bine decat daca nu am avea tool-ul asta, aici macar nu avem problema ca "on my machine it works", ca le rulam pe toate pe acelasi raspberry :)))